doc/rst/technotes/libraries.rst

```diff
@@ -591,3 +727,53 @@ supported.  We hope to extend this support in the future.
 
 C interoperability does not support default values for function arguments.  We
 do not anticipate supporting argument default values in C.
+
+.. _Python Intents:
+
+Intents in Python Interoperability
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Libraries compiled for Python do not support the default intent for ``string``,
+``c_string`` or 1D arrays, as copies are currently performed (which would
+violate ``const ref``).  Instead, only the ``in`` and ``const in`` intents are
+supported for these types.  We may be able to support all intents in the future.
+
+.. _Multilocale Caveats:
+
+Multilocale Caveats
+-------------------
+
+.. _Other Settings:
+
+Other Settings
+~~~~~~~~~~~~~~
+
+We intend to expand support for ``--library`` to include:
+
+- ``CHPL_COMM = ugni``
+- ``--no-local``
+- ``CHPL_COMM = none`` when ``CHPL_LAUNCHER != none``
+
+These settings would behave similarly to the current behavior with ``CHPL_COMM =
+gasnet``, when relevant - for instance, it is expected that all of these
+settings would result in an executable that communicates with the user's program
+via the generated library.
+
+Other situations may also become supported in the future.
+
+Supported Types
+~~~~~~~~~~~~~~~
+
+Multilocale libraries support the same argument and return types as single
+locale libraries, with the notable except of ``complex`` numbers, arrays, and
+pointer types.  We anticipate extending the supported types in the future,
+though may not end up supporting pointer types.
+
+Intents
+~~~~~~~
+
+Multilocale libraries do not support the default intent for ``string`` and
+``c_string``, as the default intent is ``const ref``.  Only the ``in`` and
+``const in`` intents are supported.  We may expand to support other intents,
+though unlike with :ref:`Python Intents`, ``ref`` and ``const ref`` seem
+unlikely.
```

Comment:
I'd leave out the bit about what seems unlikely.

Reply:
I put it in to manage expectations - the caveats section is intentionally a lot more wobbly, so it seems appropriate to distinguish between "this isn't supported because we haven't gotten to it yet" and "we kinda suspect this won't be possible"

Reply:
I'd just say, in the positive, what might be supported in the future. E.g. "We may expand to support ``inout`` and ``out`` intents in the future" (if that's the way you feel about it). I think the link to Python Intents just makes it more confusing (I had to go reread the linked section to figure out what you were talking about ).

Reply:
Okay, that's reasonable, thanks!